kohttp/src/main/kotlin/io/github/rybalkinsd/kohttp/interceptors/LoggingInterceptor.kt

```diff
@@ -24,7 +24,7 @@ class LoggingInterceptor(private val log: (String) -> Unit = ::println) : Interc
         return chain.proceed(request).also { response ->
             log("${request.method()} ${response.code()} - ${System.currentTimeMillis() - startTime}ms ${request.url()}")
 
-            request.headers().asSequence().forEach { log("${it.name}: ${it.value}") }
+            request.headers().asSequence().forEach { (name, value) ->log("$name: $value") }
```

Comment:
Data class also can be destructed, it is not necessary to make it alias for 'Pair<String, String>'

Reply:
the reason for it is `Header` data class is also removed in this PR